src/main/java/uk/gov/hmcts/reform/sscs/model/Dow.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.reform.sscs.model;
+
+import lombok.Getter;
+
+import java.util.Arrays;
+import java.util.Locale;
+
+@Getter
```

Comment:
Same getter question.

I know this might need to align with HMC but is it possible for this to be DayOfWeek instead?

Reply:
Yeah, DayOfWeek will make more then sense , I kept name same as in confluence page